### PR TITLE
Possible new features & FIxes - Needs checking

### PR DESCRIPTION
**Pull request by UntouchedOdin0**
Possible fix for double bump and a possible new feature? need someone to check it over

### CODE REVIEW

File: src/main/kotlin/net/developerden/devdenbot/commands/info/StackOverflowCommand.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    override val ddbConfig: DDBConfig,
+) : DevDenSlashCommand(
+    name = "stackoverflow",
+    description = "A helpful way to find answers using stackoverflow"
+), HasConfig {
+    override suspend fun load(action: CommandCreateAction) = Unit
+
+    override suspend fun SlashCommandEvent.execute() {
+        val question = getOption("question")?.asString ?: error("Invalid question")
+        val response = URL(question.replace("https://api.stackexchange.com/2.3/search/advanced?order=desc&sort=activity&q=$question",
```
Comment (knightzmc) [inline src/main/kotlin/net/developerden/devdenbot/commands/info/StackOverflowCommand.kt:26]: Does this work?

File: src/main/kotlin/net/developerden/devdenbot/commands/info/StackOverflowCommand.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "%20")).readText()
+        reply("Searching stackoverflow for $question")
```
Comment (knightzmc) [inline src/main/kotlin/net/developerden/devdenbot/commands/info/StackOverflowCommand.kt:28]: You need to `.await()` the `reply` calls

File: src/main/kotlin/net/developerden/devdenbot/listener/BumpNotificationListener.kt [hunk 3]
```diff
@@ -64,6 +64,7 @@ class BumpNotificationListener : EventListener {
         event.channel.sendMessage(
             "${bumpNotificationRole.asMention}, the server is ready to be bumped! **!d bump**"
         ).await()
+        return
```
Comment (knightzmc) [inline src/main/kotlin/net/developerden/devdenbot/listener/BumpNotificationListener.kt:67]: This is unnecessary, it won't fix anything

**Review by knightzmc — CHANGES_REQUESTED** [review]

**Review by UntouchedOdin0 — COMMENTED** [review]
I thought this would be cool, idk how to remove the other things sorry

**Pull request closed without merging**